app/1.0/docs/devguide/data-system.md

```diff
@@ -326,16 +326,16 @@ Related tasks:
 Polymer implements the mediator pattern, where a host element manages data flow between itself and
 its local DOM nodes.
 
-When two elements are connected with a data binding, data changes can flow *downward*, that is, from
-host to target, *upward*, from target to host, or both ways.
+When two elements are connected with a data binding, data changes can flow **downward** — from
```

Comment:
We use italics typically when introducing a new term or defining one, so I think the italics are correct here.

We don't use space around an em dash. In other words—just the dash.